File: Java-OOP-Basics/Problems Course Instance 2016_2017/src/Lesson04Polymorphism/Exercise/pr03_wild_farm/foods/Food.java
package Lesson04Polymorphism.Exercise.pr03_wild_farm.foods;

public abstract class Food {

    private int quantity;

    protected Food(int quantity) {
        this.setQuantity(quantity);
    }

    public final int getQuantity() {
        return this.quantity;
    }

    private void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}

File: C#-OOP-Basics/Problems Course Instance 2018/Polymoprhism/WildFarm/Models/Foods/Food.cs
public abstract class Food
{
    private int quantity;

    public Food(int quantity)
    {
        this.Quatity = quantity;
    }

    public int Quatity
    {
        get
        {
            return this.quantity;
        }
        set
        {
            this.quantity = value;
        }
    }
}